polar-core/src/vm.rs

```diff
@@ -735,31 +639,19 @@ impl PolarVirtualMachine {
         for (var, value) in bindings.iter() {
             self.bind(var, value.clone());
         }
-        self.csp += bindings.len();
+        self.csp = self.bsp();
```

Comment:
Was this a bug before? It was incrementing it by the bsp?

Reply:
This was only called during initialization once, so it was equivalent.